packages/manager/apps/pci-public-ip/src/pages/order/hooks/useProducts.ts

```diff
@@ -0,0 +1,33 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { useQuery } from '@tanstack/react-query';
+import { useMe } from '@/pages/order/hooks/useMe';
+
+export type TProducts = {
+  plans: {
+    planCode: string;
+    invoiceName: string;
+  }[];
+};
+
+const getUrl = (ovhSubsidiary: string) =>
+  `/order/catalog/formatted/ip?ovhSubsidiary=${ovhSubsidiary}`;
+
+export const getProducts = async (
+  ovhSubsidiary: string,
+): Promise<TProducts> => {
+  const { data } = await v6.get(getUrl(ovhSubsidiary));
+  return data as TProducts;
```

Comment:
I preffered tp pass a type the the get function  ( get<Product> ) than cast the response